MessageBroker.js

```diff
@@ -440,7 +440,7 @@ class MessageBroker {
 			}
 
 			if (msg.eventType == "custom/myVTT/token" && (msg.sceneId == window.CURRENT_SCENE_DATA.id || msg.data.id in window.TOKEN_OBJECTS)) {
-				self.handleToken(msg);
```

Comment:
This doesn't actually debounce this. Iirc something about it needing to be in a defined function. We use const in other places to define it outside the function we're using it in and it's clear it's working in sync so I assume that behaves similarly.

I don't think we want to either as otherwise it will only listen to the last token update - when moving a group of three tokens this message is sent three times but only 1 token would be listened too. I think we're ok listening to all the token events that come in - we just want to debounce sending them.

I'm approving the PR anyway though since it doesn't have an effect atm one of us can remove it later when we have time. Very nice - this feels more consistent with dragging and improves messages 🔥

Reply:
Ah this was accidental commit from when I was playing around. Thanks for catching. Im just gonna remove it real quick!